contracts/assembly/src/migration.rs

```diff
@@ -125,7 +125,10 @@ pub(crate) fn migrate_config_to_130(
     let mut cfg = Config {
```

Comment:
I think mut is not needed anymore?

Reply:
We need it to set cfg.vxastro_token_addr at 143